.github/workflows/CD.yml

```
@@ -39,25 +39,27 @@ jobs:
         echo -n "$KOREAEXIM_PEM" | base64 --decode > koreaexim.pem
 
-    - name: Copy application.yml
+    - name: Copy application-prod.yml
       env:
-        CREATE_SECRET: ${{ secrets.APPLICATION_YML }}
-        CREATE_SECRET_DIR: src/main/resources
-        CREATE_SECRET_DIR_FILE_NAME: application.yml
+        YML: ${{ secrets.APPLICATION_PROD_YML }}
+        YML_DIR: src/main/resources
+        YML_NAME: application-prod.yml
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:10: Double quote to prevent globbing and word splitting

(shellcheck)